Library/Homebrew/formula.rb

```diff
@@ -2792,8 +2799,13 @@ def deprecated?
     # <pre>disable! date: "2020-08-27", because: "has been replaced by foo"</pre>
     def disable!(date: nil, because: nil)
       odeprecated "`disable!` without a reason", "`disable! because: \"reason\"`" if because.blank?
+      odeprecated "`disable!` without a date", "`disable! date: \"#{Date.today}\"`" if date.blank?
```

Comment:
```suggestion
      odeprecated "`disable!` without a date", "`disable! date: ...`" if date.blank?
```

Just for consistency between the `disable!` and `deprecate!` deprecation messages. No preference as to whether we include the data or not but might want to be consistent.